File: core/src/SSRAG/Models/CeleryTask.cs
using SSRAG.Datory;
using SSRAG.Datory.Annotations;

namespace SSRAG.Models
{
    [DataTable("ssrag_celery_task")]
    public class CeleryTask : Entity
    {
        [DataColumn]
        public int SiteId { get; set; }

        [DataColumn]
        public int ChannelId { get; set; }

        [DataColumn]
        public int ContentId { get; set; }

        [DataColumn]
        public string TaskId { get; set; }

        [DataColumn]
        public string TaskStatus { get; set; }

        [DataColumn]
        public string TaskResult { get; set; }
    }
}

File: web/src/models/celeryTask.ts
export interface CeleryTask {
  id: number
  uuid: string
  createdDate: Date
  lastModifiedDate: Date
  siteId: number
  channelId: number
  contentId: number
  taskId: string
  taskName: string
  taskStatus: string
  taskResult: string
  resultObj: {
    id: number
    fileName: string
    extName: string
    fileSize: number
    title: string
  } | null
}
